docs/source/usersguide/install.rst

```diff
@@ -40,6 +40,26 @@ It is possible to list all of the versions of OpenMC available on your platform
 
     conda search openmc --channel conda-forge
 
+To list the versions of OpenMC that are available on the `conda-forge` channel,
+in your terminal window or an Anaconda Prompt run:
+
+.. code-block:: sh 
+
+    conda search openmc
+    
+OpenMC can then be installed with:
+
+.. code-block:: sh
+
+    conda create -n openmc-env openmc
+    
+This will install OpenMC in a conda environment called `openmc-env`. To activate
+the environment, run:
+
+.. code-block:: sh
+
+    conda activate openmc-env
+
```

Comment:
Some of this information is redundant with what appears above. I would just change this section to match what is currently in `quickinstall.rst`.

Reply:
Thanks @paulromano.